src/Jupyter/Magic/EstimateMagic.cs

```diff
@@ -114,6 +114,7 @@ public async Task<ExecutionResult> RunAsync(string input, IChannel channel)
             qsim.OnDisplayableDiagnostic += channel.Display;
             qsim.DisableLogToConsole();
 
+            channel.Stdout("The Resources Estimator will be removed in the March 2023 QDK release. Please use the Azure-based version instead.");
```

Comment:
I suggest to add a link that helps users to know how to use the Azure-based version.

Reply:
This is an approved wording. I will merge it "as is" because it is sufficient. There was a concern about putting a link in the message. We will update it if we agree on a better wording (with the link).